--- a/bazel/grpc_python_deps.bzl	2021-09-17 16:23:31.405837793 +0000
+++ b/bazel/grpc_python_deps.bzl	2021-09-17 16:24:00.282143324 +0000
@@ -53,13 +53,6 @@
             sha256 = "aa96a691d3a8177f3215b14b0edc9641787abaaa30363a080165d06ab65e1161",
         )
 
-    python_configure(name = "local_config_python")
-
-    native.bind(
-        name = "python_headers",
-        actual = "@local_config_python//:python_headers",
-    )
-
     if "cython" not in native.existing_rules():
         http_archive(
             name = "cython",
